types/react/index.d.ts

```diff
@@ -3922,6 +3924,145 @@ declare namespace React {
+        lquote?: string | undefined;
+        rquote?: string | undefined;
+    }
+    interface MathMLMspaceAttributes extends MathMLAttributes<MathMLMspaceElement> {
+        depth?: string | undefined;
+        height?: string | undefined;
+        width?: string | undefined;
+    }
+    interface MathMLMsqrtAttributes extends MathMLAttributes<MathMLMsqrtElement> {}
+    interface MathMLMstyleAttributes extends MathMLAttributes<MathMLMstyleElement> {}
+    interface MathMLMsubAttributes extends MathMLAttributes<MathMLMsubElement> {}
+    interface MathMLMsubsupAttributes extends MathMLAttributes<MathMLMsubsupElement> {}
+    interface MathMLMsupAttributes extends MathMLAttributes<MathMLMsupElement> {}
+    interface MathMLMtableAttributes extends MathMLAttributes<MathMLMtableElement> {
+        /* This attribute is non-standard. */
+        align?: string | undefined;
+        /* This attribute is non-standard. */
+        columnalign?: string | undefined;
+        /* This attribute is non-standard. */
+        columnlines?: string | undefined;
+        /* This attribute is non-standard. */
+        columnspacing?: string | undefined;
+        /* This attribute is non-standard. */
+        frame?: "none" | "solid" | "dashed" | undefined;
+        /* This attribute is non-standard. */
+        framespacing?: string | undefined;
+        /* This attribute is non-standard. */
+        rowalign?: string | undefined;
+        /* This attribute is non-standard. */
+        rowlines?: string | undefined;
+        /* This attribute is non-standard. */
+        rowspacing?: string | undefined;
+        /* This attribute is non-standard. */
+        width?: string | undefined;
+    }
+    interface MathMLMtdAttributes extends MathMLAttributes<MathMLMtdElement> {
+        /* This attribute is non-standard. */
+        columnalign?: "left" | "center" | "right" | undefined;
+        columnspan?: number | string | undefined;
+        /* This attribute is non-standard. */
+        rowalign?: "axis" | "baseline" | "bottom" | "center" | "top" | undefined;
+        rowspan?: number | string | undefined;
+    }
+    interface MathMLMtextAttributes extends MathMLAttributes<MathMLMtextElement> {}
+    interface MathMLMtrAttributes extends MathMLAttributes<MathMLMtrElement> {
+        /* This attribute is non-standard. */
+        columnalign?: "left" | "center" | "right" | undefined;
+        /* This attribute is non-standard. */
+        rowalign?: "axis" | "baseline" | "bottom" | "center" | "top" | undefined;
+    }
+    interface MathMLMunderAttributes extends MathMLAttributes<MathMLMunderElement> {
+        accentunder?: boolean | undefined;
+    }
+    interface MathMLMunderoverAttributes extends MathMLAttributes<MathMLElement> {
+        accent?: boolean | undefined;
+        accentunder?: boolean | undefined;
+    }
+    /**
+     * @see https://w3c.github.io/mathml-core/#semantics-and-presentation
```

Comment:
This and the urls below should be `link`s like used elsewhere for better in-editor readability.